src/styles/card.scss

```diff
@@ -56,11 +66,13 @@
     justify-content: center;
     align-items: center;
     text-align: center;
+    transition: transform 500ms;
 
     &:hover {
       background-color: white;
       color: $main__color;
       border: 1px solid $main__color;
+      transform: scale(1.1);
```

Comment:
It should increase by 20 percent, not 10.

![image](https://user-images.githubusercontent.com/60885495/113719219-0d0cdc80-96f6-11eb-9c1f-9e4b4c558815.png)